Yosemite/YosemiteTests/Stores/AppSettingsStoreTests.swift

```diff
@@ -519,6 +519,127 @@ final class AppSettingsStoreTests: XCTestCase {
         XCTAssertTrue(isEnabled)
     }
 
+    func test_loadJetpackBenefitsBannerVisibility_returns_true_on_new_generalAppSettings() throws {
+        // Given
+        try fileStorage?.deleteFile(at: expectedGeneralAppSettingsFileURL)
+
+        let currentTime = Date(timeIntervalSince1970: 1638105924)
+        let calendar = Calendar(identifier: .gregorian)
+
+        // When
+        let isVisible: Bool = waitFor { promise in
+            let action = AppSettingsAction.loadJetpackBenefitsBannerVisibility(currentTime: currentTime, calendar: calendar) { isVisible in
+                promise(isVisible)
+            }
+            self.subject?.onAction(action)
+        }
+
+        // Then
+        XCTAssertTrue(isVisible)
+    }
+
+    func test_loadJetpackBenefitsBannerVisibility_returns_true_after_setting_last_dismissed_date_exactly_five_days_ago_without_dst() throws {
+        // Given
+        try fileStorage?.deleteFile(at: expectedGeneralAppSettingsFileURL)
+
+        // GMT - Tuesday, November 23, 2021 1:25:24 PM
+        let lastDismissedTime = Date(timeIntervalSince1970: 1637673924)
+        // GMT - Sunday, November 28, 2021 1:25:24 PM - exactly five days after the last dismissed date without DST
+        let currentTime = Date(timeIntervalSince1970: 1638105924)
+        let calendar: Calendar = {
+            var calendar = Calendar(identifier: .gregorian)
+            guard let timeZoneWithoutDailySavingTime = TimeZone(identifier: "Asia/Taipei") else {
```

Comment:
Super nit: Are we referring to Daylight Saving Time as "DailySavingTime" here? I am wondering whether the term "daylight saving time" is valid. 🤔 
Please ignore if I am wrong. 😅

Reply:
Good catch! I think it's just a typo 😛

Reply:
Great catch on the typo @selanthiraiyan! I'm not sure why I was thinking "daily saving time" 😅

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5505/commits/5d82aedc78b7120b7d8d1845750712195832e9d7